src/xtopcom/xvledger/src/xvblock.cpp

```diff
@@ -2304,6 +2290,16 @@ namespace top
             
             return xdataobj_t::query_interface(_enum_xobject_type_);
         }
+
+        uint64_t xvblock_t::get_second_level_gmtime() const
+        {
+            // second_level_gmtime used for transaction consensus time statistic
+            uint64_t gmtime = get_header()->get_second_level_gmtime();
+            if (0 == gmtime) {
+                gmtime = get_cert()->get_clock_level_gmtime();
```

Comment:
why return **clock**_level_gmtime in get_**second**_level_gmtime ?

Reply:
add explanation